test/integration/document/PresentationViewer.e2e.test.js

```diff
@@ -0,0 +1,34 @@
+// <reference types="Cypress" />
+describe('Presentation Viewer', () => {
+    const token = Cypress.env('ACCESS_TOKEN');
+    const fileId = Cypress.env('FILE_ID_PRESENTATION');
+
+    /* eslint-disable */
+    const showControls = () => {
```

Comment:
I think this is duplicated in other test files. Should it be pulled out as a top-level helper?

Reply:
done